packages/backend/src/boot/master.ts

```diff
@@ -110,8 +107,9 @@ function showNodejsVersion(): void {
 
 	nodejsLogger.info(`Version ${runningNodejsVersion.join('.')}`);
 
-	if (!satisfyNodejsVersion) {
-		nodejsLogger.error(`Node.js version is less than ${requiredNodejsVersion.join('.')}. Please upgrade it.`, null, true);
+	const minVersion = 'v16.14.0';
```

Comment:
I think it would be a good idea to read the version from the `.node-version` file, otherwise we could end up in the same situation again where only one of the two locations was maintained.

```suggestion
	const minVersion = fs.readFileSync('${_dirname}/../../../../.node-version', 'utf-8').trim();
```